demisto_sdk/commands/init/initiator.py

```diff
@@ -222,7 +222,15 @@ def convert_contribution_to_pack(self):
                     dst_name = ENTITY_TYPE_TO_DIR.get(basename)
                     src_path = os.path.join(pack_dir, basename)
                     dst_path = os.path.join(pack_dir, dst_name)
-                    content_item_dir = shutil.move(src_path, dst_path)
+                    if os.path.exists(dst_path):
```

Comment:
Perhaps add a comment here about the case where this applies if you think it would be helpful. Up to you - otherwise, it looks good.

Reply:
I thought about adding in an explanation about why we do this, but I feel the added comments explain what's being done, and though it doesn't explain the full reason why, it does enough to warrant the user (while the UT add another layer of warranty)